cinnabar/measurements.py

```diff
@@ -52,19 +54,52 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        IC50: float,
                         label: str,
+                        uncertainty: float = 0,
                         source: str = '',
+                        temperature: float = 298,
                         ):
         """Create Measurement from experimental data
 
         Parameters
         ----------
+        IC50: float
+          experimental IC50 value in unit nM
+        label: str, optional
+          label for this data point.
+        uncertainty: float, optional
+          uncertainty of the experimental value in nM, default is zero if no uncertainty is provided
+        source: str, optional
+          source of experimental measurement
+        temperature: float
+          temperature in K at which the experimental measurement was carried out. By default: 298 K
         """
+        R = 1.9872042586408 * 0.001 * unit.kilocalorie_per_mole / unit.kelvin # Gas constant in kcal/mol/K
+        IC50_upper = IC50 * unit.nanomolar + uncertainty * unit.nanomolar
+        IC50_lower = IC50 * unit.nanomolar - uncertainty * unit.nanomolar
```

Comment:
we can tweak this now we've looked up the proper way.  maybe include the link to the solution we found so it's obvious what we're doing here